### PR TITLE
update docs and add btcDisplayAddressSimple return address

### PR DESCRIPTION
**Pull request by thisconnect**
- added support for display=false for `btcDisplayAddressSimple` and `ethDisplayAddress`
- update documentation on methods.md
- changed demo.js to first get an address with display=false, show the address and then ask to confirm the address on the bitbox

### CODE REVIEW

File: demo/demo.js [hunk 1]
```diff
@@ -141,11 +141,12 @@ ethSignMsg.addEventListener("click", async () => {
 
 // Display single-sig BTC address on device for verification
 document.getElementById("btcAddressSimple").addEventListener("click", async () => {
-    await device.api.btcDisplayAddressSimple(
+    const address = await device.api.btcDisplayAddressSimple(
         constants.messages.BTCCoin.BTC,
         getKeypathFromString("m/49'/0'/0'/0/0"),
         constants.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
     );
+    alert("btcDisplayAddressSimple returned:\n" + address);
```
Comment (benma) [inline demo/demo.js:149]: What you could do for a more natural workflow:

- get address without displaying (display param = false)
- show  the address
- display address on BitBox02

Comment (thisconnect) [reply demo/demo.js]: PTAL https://github.com/digitalbitbox/bitbox02-api-js/pull/40/commits/f4f2d0d9fa2a4b92a9b2aeeb9c6e7805887f26dd

**Review by benma — APPROVED** [review]
utACK.

What about doing the same for ETH?

**Comment by thisconnect** [comment]
> What about doing the same for ETH?

added please take a look, I am not sure if there will be other args on `ethDisplayAddress(keypath, display = true)` if yes I'd like to have them first, so `display` is the last arg.

**Pull request merged**